build.xml

```diff
@@ -38,6 +38,7 @@
     <property name="javac.target" value="1.8"/>
     <property name="javac.debug" value="true"/>
 
+    <property name="includedTestGroups" value=""/>
```

Comment:
Can you make this a param in the `antcall` for the `test` target? It is confusing to have an empty value defined up here like this.